CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 **v5.6.0(unreleased):**
 
+* Updated Otter Grade CSV to indicate which notebooks timeout [#813](https://github.com/ucbds-infra/otter-grader/issues/813)
```

Comment:
nit: add "per" in front of the issue link; i.e. `notebooks timeout per [#813](https://github.com/ucbds-infra/otter-grader/issues/813)`